en/monitoring/monitoring-servers/add-a-poller-to-configuration.md

```diff
@@ -63,6 +63,11 @@ recommended).
 
 Click on **Save**.
 
+> Note that the *SSH Legacy port* is not used anymore and will be removed.
```

Comment:
This notice should be in the "Using SSH" tab

Reply:
This legacy port is not the Gorgone port if Gorgone is in SSH mode.
It was the one used by Centcore and Service Discovery, and is not used anymore, except maybe by custom user scripts.